src/test/java/org/fundacionjala/coding/dennis/Twist37Test.java

```diff
@@ -0,0 +1,64 @@
+package org.fundacionjala.coding.dennis;
+
+import org.junit.Test;
+import static org.junit.Assert.assertEquals;
+import java.util.Arrays;
+
+/**
+ *
+ * @author DennisMont
+ *
+ */
+public class Twist37Test {
+
+    public static final int ONE = 1;
+    public static final int TWO = 2;
+    public static final int THREE = 3;
+    public static final int FOUR = 4;
+    public static final int FIVE = 5;
+    public static final int SIX = 6;
+    public static final int SEVEN = 7;
+    public static final int EIGHT = 8;
+    public static final int NINE = 9;
+    public static final int TWELVE = 12;
+    public static final int FOURTEEN = 14;
+    public static final int THIRTEEN = 13;
```

Comment:
@AT-08/at-08 
Don't create constants for each number.
You should put the int array as final